File: APIRails/app/models/clan.rb
class Clan < ApplicationRecord
  belongs_to :game
  has_many :game_users, dependent: :destroy
  has_many :users, through: :game_users

  validates :color, presence: true
  validates :name, presence: true
  validates :start_q, presence: true, numericality: { only_integer: true }
  validates :start_r, presence: true, numericality: { only_integer: true }
end 

File: APILaravel/app/Models/Clan.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Clan extends Model
{
    protected $fillable = [
        'game_id',
        'color',
        'name',
        'start_q',
        'start_r',
    ];

    public function game()
    {
        return $this->belongsTo(Game::class);
    }

    public function gameUsers()
    {
        return $this->hasMany(GameUser::class);
    }

    public function users()
    {
        return $this->belongsToMany(User::class, 'game_users');
    }
}
